README.md

```diff
@@ -12,7 +12,7 @@ This [dbt](https://www.getdbt.com/) project has a split personality:
 
 - **Tutorial**: The [tutorial](https://github.com/fishtown-analytics/jaffle_shop/tree/master)
```

Comment:
major overhaul of the readme for cleaner instructions, coherent flow, and additional functionality